sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -44,14 +44,17 @@ function getTopicFilter(value: any): SqlRuleFilter | CorrelationRuleFilter {
   } else {
     result = {
       correlationId: getStringOrUndefined(value["CorrelationId"]),
-      label: getStringOrUndefined(value["Label"]),
+      subject: getStringOrUndefined(value["Label"]),
       to: getStringOrUndefined(value["To"]),
       replyTo: getStringOrUndefined(value["ReplyTo"]),
       replyToSessionId: getStringOrUndefined(value["ReplyToSessionId"]),
       sessionId: getStringOrUndefined(value["SessionId"]),
       messageId: getStringOrUndefined(value["MessageId"]),
       contentType: getStringOrUndefined(value["ContentType"]),
-      properties: getKeyValuePairsOrUndefined(value["Properties"], "UserProperties")
+      applicationProperties: getKeyValuePairsOrUndefined(
+        value["ApplicationProperties"],
```

Comment:
`value` here refers to the value returned by the service which would continue to be `Properties` because the service is not being changed. This is why few live tests are failing.

```suggestion
        value["Properties"],
```